README.md

```diff
@@ -11,92 +11,131 @@ npm install rn-firebase-chat
-  avatar: 'https://example.com/avatar.jpg'
-}
+  avatar: 'https://example.com/avatar.jpg',
+};
 
 function App() {
-
   return (
     <ChatProvider userInfo={userInfo}>
-      <AppNavigation/>
+      <AppNavigation />
     </ChatProvider>
   );
 }
 ```
+
 - Setup navigation for `ListConversationScreen` and `ChatScreen`
 
 ```javascript
 export const ChatNavigator = () => (
   <Stack.Navigator>
-    <Stack.Screen
-      name={RouteKey.ListChatScreen}
-      component={ListChatScreen}
-    />
-    <Stack.Screen
-      name={RouteKey.ChatScreen}
-      component={ChatScreen}
-    />
+    <Stack.Screen name={RouteKey.ListChatScreen} component={ListChatScreen} />
+    <Stack.Screen name={RouteKey.ChatScreen} component={ChatScreen} />
   </Stack.Navigator>
-)
+);
 ```
 
 ```javascript
-import React, {useCallback} from 'react';
-import {ListConversationScreen} from 'rn-firebase-chat';
-import {navigate} from '../../navigation/NavigationService';
+import React, { useCallback } from 'react';
+import { ListConversationScreen } from 'rn-firebase-chat';
+import { navigate } from '../../navigation/NavigationService';
 import RouteKey from '../../navigation/RouteKey';
 
 export const ListChatScreen: React.FC = () => {
   const handleItemPress = useCallback((data) => {
     navigate(RouteKey.ChatScreen, data);
   }, []);
 
+  return <ListConversationScreen onPress={handleItemPress} />;
+};
+```
+
+```javascript
+import React from 'react';
+import { ChatScreen as BaseChatScreen } from 'rn-firebase-chat';
+
+const partnerInfo = {
+  id: 'ayz123',
+  name: 'Tony',
+  avatar: 'https://example.com/tony.jpg',
+};
+
+export const ChatScreen: React.FC = () => {
   return (
-      <ListConversationScreen onPress={handleItemPress} />
+    <BaseChatScreen memberIds={[partnerInfo.id]} partners={[partnerInfo]} />
   );
 };
+```
+
+## Addons
+
+Additional features for chat are:
+
+#### Image/Video Picker and Camera
+
+This feature will require additional libraries:
+
+- Using [npm](https://www.npmjs.com/#getting-started):
+
+```sh
+npm install react-native-fast-image react-native-video react-native-vision-camera uuid react-native-image-picker --save
+```
 
+- Using [Yarn](https://yarnpkg.com/):
+
+```sh
+yarn add react-native-fast-image react-native-video react-native-vision-camera uuid react-native-image-picker
 ```
 
+Then using our Addons component in ChatScreen
+
 ```javascript
 import React from 'react'
 import {ChatScreen as BaseChatScreen} from 'rn-firebase-chat'
+import {CameraView, useCamera} from 'rn-firebase-chat/src/addons/camera'
 
-const partnerInfo = {
-  id: 'ayz123',
-  name: 'Tony',
-  avatar: 'https://example.com/tony.jpg'
-}
+...
 
 export const ChatScreen: React.FC = () => {
+  const {onPressCamera, onPressGallery} = useCamera()
   return (
     <BaseChatScreen
       memberIds={[partnerInfo.id]}
       partners={[partnerInfo]}
-    />
+      hasCamera={true}
+      hasGallery={true}
+      inputToolbarProps={{
+        onPressCamera,
```

Comment:
Do you think we should move hasCamera, hasGallery into inputToolbarProps?

Reply:
moved